test/metabase/util/date_2_test.clj

```diff
@@ -250,7 +261,6 @@
             :hour     (t/zoned-date-time  2019 10 27 14 0  0 0 (t/zone-id "America/Los_Angeles"))
             :day      (t/zoned-date-time  2019 10 27  0 0  0 0 (t/zone-id "America/Los_Angeles"))
             :week     (t/zoned-date-time  2019 10 27  0 0  0 0 (t/zone-id "America/Los_Angeles"))
-            :iso-week (t/zoned-date-time  2019 10 21  0 0  0 0 (t/zone-id "America/Los_Angeles"))
```

Comment:
i'm not sure if i know enough for this question to make sense, but is it possible any cards or queries or questions or what-have-yous have `:iso-week` in them such that we need to support the offset still?

Reply:
We've never used `:iso-week` or the other ones I removed anywhere outside of the date util namespace. When I wrote that date util namespace we only supported Sunday-start weeks, so I put those `:iso-` units were in there mostly to make implementing ISO week support easier when we actually got around to it. We didn't end up implementing ISO week support -- we implemented the more general `start-of-week` Setting, so `:iso-week` and the like never ended up being used anywhere.

Anyways the valid units in Cards/queries/etc. are all defined in `metabase.mbql.schema` and regardless of what's in here you can't use anything not in the MBQL schema. The `:iso-` options were never in there